File: client/src/components/InventoryTransaction.js
const InventoryTransaction = ({
  activity,
  quantityBefore,
  quantityAfter,
  date,
  inventory: { name },
}) => {
  return (
    <div className="transaction">
      <div>{date.substring(0, 10)}</div>
      <div>{name}</div>
      <div>{activity}</div>
      <div>{quantityBefore}</div>
      <div>{quantityAfter}</div>
    </div>
  );
};

export default InventoryTransaction;

File: Model/InventoryTransaction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Inventory_Management.Model
{
    public class InventoryTransaction
    {
        public int Id { get; set; }

        public int InventoryId { get; set; }

        public decimal Price { get; set; }

        public Inventory Inventory { get; set; }

        public ActivityClass Activity { get; set; }

        public int QuantityBefore { get; set; }

        public int QuantityAfter { get; set; }

        public DateTime Date { get; set; } = DateTime.Today;


    }
}